datamol/viz/_lasso_highlight.py

```diff
@@ -0,0 +1,439 @@
+# This file is thanks Christian W. Feldman
+# Christian Feldman (2021) lassohighlight [sourcecode]. https://github.com/c-feldmann/lassohighlight.
+from rdkit.Chem.Draw import rdMolDraw2D
+from rdkit.Chem.rdmolops import Get3DDistanceMatrix
+from rdkit import Chem
+from rdkit.Geometry.rdGeometry import Point2D
+import numpy as np
+from collections import defaultdict
+from collections import namedtuple
+from typing import *
+from PIL import Image
+import io
+from loguru import logger
+
+import datamol as dm
+from .utils import prepare_mol_for_drawing
+
+
+def _angle_to_coord(center: np.ndarray, angle: np.float64, radius: np.float64) -> np.ndarray:
+    """Determines a point relative to the center with distance (radius) at given angle.
+    Angles are given in rad and 0 rad correspond to north of the center point.
+    """
+
+    x = radius * np.sin(angle)
+    y = radius * np.cos(angle)
+    x += center[0]
+    y += center[1]
+    return np.array([x, y])
+
+
+def _arch_points(
+    radius: np.float64, start_ang: np.float64, end_ang: np.float64, n: int
```

Comment:
Replace all `np.float64` type by `float` (this is what usually a user will use).